traffic_ops/install/bin/postinstall.test.sh

```diff
@@ -0,0 +1,424 @@
+	print('Malformed cdn.conf file - not an object or missing keys', file=sys.stderr)
+	exit(1)
+
+if not isinstance(conf['hypnotoad'], dict) or len(conf['hypnotoad']) != 1 or 'listen' not in conf['hypnotoad'] or not isinstance(conf['hypnotoad']['listen'], list) or len(conf['hypnotoad']['listen']) != 1 or not insinstance(conf['hypnotoad']['listen'][0], str):
+	print('Malformed hypnotoad object in cdn.conf:', conf['hypnotoad'], file=sys.stderr)
+	exit(1)
+
+listen = 'https://[::]:6443?cert=$ROOT_DIR/etc/pki/tls/certs/localhost.crt&key=$ROOT_DIR/etc/pki/tls/private/localhost.key'
+if conf['hypnotoad']['listen'][0] != listen:
+	print('Incorrect hypnotoad.listen[0] in cdn.conf, expected:', listen, 'got:', conf['hypnotoad']['listen'][0], file=sys.stderr)
+	exit(1)
+
+if not isinstance(conf['secrets'], list) or len(conf['secrets']) != 1 or not isinstance(conf['secrets'][0], str):
+	print('Malformed secrets object in cdn.conf:', conf['secrets'], file=sys.stderr)
+	exit(1)
+
+if len(conf['secrets'][0]) != 12 or any(True for x in conf['secrets'][0] if x not in string.ascii_letters + string.digits + '_'):
+	print('Incorrect secret in cdn.conf, expected 12 word characters, got:', conf['secrets'][0], file=sys.stderr)
+	exit(1)
+
+if not isinstance(conf['to'], dict) or 'base_url' not in conf['to'] or len(conf['to']) != 1 or not isinstance(conf['to']['base_url'], str):
+	print('Malformed to object in cdn.conf:', conf['to'])
+	exit(1)
+
+if conf['to']['base_url'] != 'http://localhost:3000':
+	print('Incorrect to.base_url in cdn.conf, expected: http://localhost:3000, got:', conf['to']['base_url'], file=sys.stderr)
+	exit(1)
+
+if not isinstance(conf['traffic_ops_golang'], dict) or len(conf['traffic_ops_golang']) != 3 or 'port' not in conf['traffic_ops_golang'] or 'log_location_error' not in conf['traffic_ops_golang'] or 'log_location_event' not in conf['traffic_ops_golang']:
+	print('Malformed traffic_ops_golang object in cdn.conf:' conf['traffic_ops_golang'], sys.stderr)
+	exit(1)
+
+if conf['traffic_ops_golang']['port'] != '443':
+	print('Incorrect traffic_ops_golang.port, expected: 443, got:', conf['traffic_ops_golang']['port'], file=sys.stderr)
+	exit(1)
+
+if conf['traffic_ops_golang']['log_location_error'] != '$ROOT_DIR/var/log/traffic_ops/error.log':
+	print('Incorrect traffic_ops_golang.log_location_error in cdn.conf, expected: $ROOT_DIR/var/log/traffic_ops/error.log, got:', conf['traffic_ops_golang']['log_location_error'], file=sys.stderr)
+	exit(1)
+
+if conf['traffic_ops_golang']['log_location_event'] != '$ROOT_DIR/var/log/traffic_ops/access.log':
+	print('Incorrect traffic_ops_golang.log_location_event in cdn.conf, expected: $ROOT_DIR/var/log/traffic_ops/access.log, got:', conf['traffic_ops_golang']['log_location_event'], file=sys.stderr)
+	exit(1)
+
+exit(0)
+";
+
+/usr/bin/python3 -c "#CDN_CONF_TEST";
+
+
+DATABASE_CONF_EXPECTED='{
```

Comment:
`DATABASE_CONF_EXPECTED` should not be all caps because it is not an environment variable.